bucketeer2/src/main/kotlin/io/bucketeer/sdk/android/internal/user/UserHolder.kt

```diff
@@ -0,0 +1,34 @@
+package io.bucketeer.sdk.android.internal.user
+
+import io.bucketeer.sdk.android.BKTUser
+import io.bucketeer.sdk.android.internal.model.User
+
+class UserHolder(
+  private var user: User,
+) {
+  val userId: String = user.id
+
+  fun get(): User = user
+
+  fun update(user: User) {
```

Comment:
Before v2, we set the user via `setUser` after calling the init interface to set the `userHolder`. 
But to simplify the init process, we decided to pass the user info via the `initialize` interface. 

To update the user, the app will need to destroy the current client instance, recreate it, and pass the new user.
Maybe we could remove it if there is no case for updating it.

Please let me know your thoughts.

Reply:
Sounds good. 
done in [`5eb44bf` (#68)](https://github.com/ca-dp/bucketeer-android-sdk/pull/68/commits/5eb44bfe07f1bfa2ed5b30b65e2dde3d707d54f2)

Reply:
Thanks!